docs/hardware/devices/rak-wireless/wisblock/peripherals.mdx

```diff
@@ -93,7 +93,7 @@ The RAK1910 is supported on the following base boards & slots:
 ## Buzzer
 
 The [RAK18001 Buzzer Module](https://store.rakwireless.com/products/wisblock-buzzer-module-rak18001) is currently being tested for integration with the External Notifications plugin. There is currently a known conflict with buzzer if the module is placed in Slot D, although other slots should work.
-
+When using the buzzer in Slot C of RAK19007 Base Board with the RAK4631(NRF Core Module), The configuration for the module is set as pin 21 in the Device section and set buzzer pin to 21 in the External Notifications module as well as use PWM.
```

Comment:
I feel that for this specific board that is is the only compatible stot for the module since it you have the GPS module, normally it is installed in slot B (because the module is double height it also covers slot A) and being that slot D is not compatible with the module buzzer it only leaves slot C, hence the information on what pin to set in the confg to allow the buzzer to be used.